firebase-project/src/views/register.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="form-container">
+    <h1>Create a User</h1>
+    <form @submit.prevent="handleSignup">
+      <label>First name</label>
```

Comment:
@mukeshsin Can you implement https://vee-validate.logaretm.com/v4/ vee-validate in all your forms